src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  let totalCost = days * dayCost;
```

Comment:
Use 'const' instead of 'let' for 'totalCost' since it is not being reassigned, but rather its value is being updated based on conditions.